scl-extension/src/main/resources/xsd/SCL_CoMPAS.xsd

```diff
@@ -83,25 +81,6 @@ SPDX-License-Identifier: Apache-2.0
         </xs:restriction>
     </xs:simpleType>
 
-    <xs:simpleType name="tCompasLabel">
```

Comment:
@pascalwilbrink do you know if we use this in the SCL dataservice?

Reply:
ok, I reverted this change